scipy/stats/_stats_py.py

```diff
@@ -1339,7 +1349,7 @@ def describe(a, axis=0, ddof=1, bias=True, nan_policy='propagate'):
     n = a.shape[axis]
     mm = (np.min(a, axis=axis), np.max(a, axis=axis))
     m = np.mean(a, axis=axis)
-    v = np.var(a, axis=axis, ddof=ddof)
+    v = _var(a, axis=axis, ddof=ddof)
```

Comment:
> We could also optimize a bit by using the mean, which already needs to be calculated outside of `_var`. (The difficulty with that is that we need the mean with `keepdims=True`.)

I guess we can just do:

```suggestion
    v = _var(a, mean=np.expand_axis(m, axis=axis), axis=axis, ddof=ddof)
```

Reply:
It doesn't make a big difference though.

Reply:
I guess I can just add the check like there is in `_moment`, then calculate the variance with `np.var`. Do you think that's the way to go?

Reply:
Yes, sounds good!

Reply:
That doesn't help : / The check itself is what is taking most of the time.

Reply:
We can leave it as it is. I don't think the slowdown is too bad.